app/src/main/java/org/fossasia/openevent/general/LoginActivity.kt

```diff
@@ -70,4 +66,12 @@ class LoginActivity : AppCompatActivity() {
         super.onDestroy()
         compositeDisposable.dispose()
     }
+
+    private fun loginEnabled(boolean:Boolean) {
```

Comment:
enabled: Boolean

Reply:
Rename to showProgress

Naming of functions is very important and needs to reflect their major task